stories/12-Accordion.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {Fragment} from "react";
+import {storiesOf} from "@storybook/react";
+import Accordion from "../src/accordion/Accordion";
+
+storiesOf("Accordion", module).add("Accordion", () => (
+  <Fragment>
+    <p>
+      {
+        'Accordion with defaultExpanded={["1"]} allowZeroExpanded={false} allowMultipleExpanded={false}'
+      }
+    </p>
+    <Accordion defaultExpanded={["1"]}>
+      <Accordion.Item accordionId="1" header="Accordion header 1">
+        Content of the Accordion 1
+      </Accordion.Item>
```

Comment:
Also, it should be a javascript string expression. Actually our linter packages should fix these, weird 🤔 
```suggestion
      <Accordion.Item accordionId={"1"} header={"Accordion header 1"}>
        {"Content of the Accordion 1"}
      </Accordion.Item>
```